### DISCUSSION TITLE
Tests in examples

### DISCUSSION THREAD

**Discussion by suprsidr** [lingui/js-lingui#1769] [General]
I wish your examples had tests. I have been able to configure jest for Vite using this [guide](https://medium.com/@jelena.durovic/how-to-configure-jest-with-typescript-fe034ad7d6c2) but the addition of `<Trans />` breaks every test.
```
Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined.
```

An example of the proper jest.config w/ vite and lingui would be helpful.

**Comment by timofei-iatsenko** [comment]
You need to setup Jest the same way as your build pipeline. If you use SWC + swc plugin it should be in the Jest as well, the same for babel with babel config.
 If you use Vite, probably Vitest would be much easier to setup because it should automatically pick up settings from your vite config
